### PR TITLE
build: use CLI rc.8

### PR DESCRIPTION
**Pull request by filipesilva**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by josephperrott — APPROVED** [review]
LGTM

**Review by gkalpak — APPROVED** [review]

**Review by alan-agius4 — CHANGES_REQUESTED** [review]
Can you also update the dependency section in the root package.json?

There are several devkit and schematics packages which are not updated.

https://github.com/angular/angular/blob/10dc0b437d4760fac7f5d68f47f87ac72ff1a704/package.json#L38-L51

Thanks.

**Comment by filipesilva** [comment]
@alan-agius4 done, good catch!

**Review by alan-agius4 — APPROVED** [review]
Thanks, LGTM.

File: package.json [hunk 1]
```diff
@@ -130,8 +130,8 @@
   "// 2": "devDependencies are not used under Bazel. Many can be removed after test.sh is deleted.",
   "// 3": "when updating @bazel/bazel version you also need to update the RBE settings in .bazelrc (see https://github.com/angular/angular/pull/27935)",
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.900.0-rc.4",
-    "@angular/cli": "9.0.0-rc.4",
+    "@angular-devkit/build-angular": "^0.900.0-rc.8",
+    "@angular/cli": "^9.0.0-rc.8",
```
Comment (clydin) [inline package.json]: These were pinned previously.  Was the change intentional?

File: integration/cli-hello-world-ivy-i18n/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "update-webdriver": "webdriver-manager update --gecko=false --standalone=false $CI_CHROMEDRIVER_VERSION_ARG",
     "start": "ng serve",
     "pretest": "ng version",
-    "test": "ng e2e --prod && ng xi18n && yarn translate && ng build --configuration production,fr && ng build --configuration production,de && run-p -r \"serve dist/fr\" \"ng e2e --configuration=fr\" && run-p -r \"serve dist/de\" \"ng e2e --configuration=de\"",
+    "test": "ng e2e --prod && ng xi18n && yarn translate && ng e2e --configuration fr && ng e2e --configuration de",
```
Comment (IgorMinar) [inline integration/cli-hello-world-ivy-i18n/package.json]: so much cleaner! thanks!

File: package.json (hunk 1 above)
Comment (IgorMinar) [reply package.json]: they should remain pinned, we don't want to take chance with various package versions getting out of sync.

**Review by IgorMinar — APPROVED** [review]
lgtm, except for the pinning. thank you for the PR!

File: package.json (hunk 1 above)
Comment (filipesilva) [reply package.json]: Changed back, my bad.

**Pull request closed without merging**